docs/Release_Notes.md

```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
+ |   exwafs_grib.sh                |<- exgfs_atmos_wafs_grib.sh                                  |
+ |   exwafs_gfs_manager.sh (new)   |   exgfs_atmos_wafs_blending.sh (removed after cleaning up)  |
+ |   exwafs_upp.sh (new)           |                                                             |
+
+Additionally there are other changes:
+1. In exwafs_grib2_0p25_blending.sh, extend the waiting time window for UK data from 15 to 25 minutes
+2. In exwafs_grib2_0p25.sh, only include fields at the extra levels when forecast hour is between 06 and 36 per AWC request
+3. In ush/mkwfsgbl.sh: change input dependency from GFS pgrb2.1p00 to master file.
+4. Remove files under ush/ folder: wafs_blending.sh wafs_grib2.regrid.sh wafs_intdsk.sh
 
 
 Fix Changes
 -----------
-
+1. Remove fix/wafs/legend folder
+2. Remove files under fix/wafs:
+   - grib2_blended_wafs_wifs_fFF.0p25
+   - grib2_gfs_wafs_wifs_fFF.0p25
+   - grib_wafsgfs_intdsk
+   - grib_wafsgfs_intdskf00   
+3. Under fix/wafs, filenames are changed according to EE2 standards.
+   - faa_gfsmaster.grb2.list   → grib2_gfs_awf_master.list
+   - gfs_master.grb2_0p25.list → grib2_0p25_gfs_master2d.list
+   - gfs_wafs.grb2_0p25.list   → grib2_0p25_wafs_hazard.list
+   - grib2_gfs_awffFF.45       → grib2_gfs_awffFFF.45 (FF FFF - forecast hours)
+   - grib2_gfs_wafsfFF.45      → grib2_wafsfFFF.45 (FF FFF - forecast hours)
+   - wafs_0p25_admin_msg       → wafs_blending_0p25_admin_msg
+   - wafs_gfsmaster.grb2.list  → grib2_wafs.gfs_master.list
+   - wafs.namelist             → grib_wafs.namelist
 
 Module Changes
 --------------
-
+No change
 
 Changes to File Sizes
 ---------------------
-
+* UPP: increased by 40G (new, moved from GFS to WAFS)
+* GRIB2_0P25: increased by 0.1G
 
 Environment and Resource Changes
 --------------------------------
-
+1. Add ecFlow to WAFS package
+2. Add UPP as a WAFS component
+3. Get rid of MPMD, each forecast hour will be run in its own job card.
+4. WAFS_GRIB job dependency is changed from GFS pgrb2.1p00 to GFS master file
 
 Pre-implementation Testing Requirements
 ---------------------------------------
+* Which production jobs should be tested as part of this implementation?
+  * The entire WAFS v7.0.0 package needs to be installed and tested on WCOSS-2
+* Does this change require a 30-day evaluation?
+  * No
+
+
+Product Changes
+---------------
+* Directory changes
+  * From com/gfs/v16.3/gfs.YYYYMMDD/CC/atmos to com/wafs/v7.0/wafs.YYYYMMDD/CC
+  * Inside WAFS, there are subfolders categoried by job names
+    * |-- upp
+    * |-- gcip
+    * |-- grib
+    * |-- grib2
+    * |----- 1p25
+    * |----- 0p25
+    * |-------- blending
+* Files to be retired
+  * `gfs.tCCz.wafs_icao.grb2fFFF`
+* Filename changes
+  * Renamed according to EE2 implementation standards
+  * Exceptions: files sent to UK
+  * Refer to this document: https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY
```

Comment:
IMO, the name changes should be summarized in the release notes.  The google document can change, and will need access updated for anyone who looks at this release note.